packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -491,11 +491,13 @@ module.exports = {
     'spaced-comment': ['error', 'always', {
       line: {
         exceptions: ['-', '+'],
-        markers: ['=', '!'], // space here to support sprockets directives
+        // support sprockets directives
```

Comment:
please move the comments back to being inline; line length isn't worth worrying about.